update: runOnIdle() no longer calls waitForIdle() after action (#323)
JetBrains/kotlin-multiplatform-dev-docs: topics/whats-new/whats-new-compose-180.md

}
```

### Implementation of `runOnIdle()` aligned with Android

To bring the Compose Multiplatform implementation of the `runOnIdle()` test function in line with Android behavior,
we introduced the following changes:

> * `runOnIdle()` now executes its `action` in the UI thread.

Reviewer: ```suggestion
* `runOnIdle()` now executes its `action` on the UI thread.
```

* `runOnIdle()` does not call `waitForIdle()` after executing `action` anymore.

> If your tests rely on that extra `waitForIdle()` call after the `runOnIdle()` action,
> add that call yourself as needed when updating tests for Compose Multiplatform %composeEapVersion%.

Reviewer: ```suggestion
If your tests rely on that extra `waitForIdle()` call after the `runOnIdle()` action,
you will need to add that call manually when updating tests for Compose Multiplatform %composeEapVersion%.
```

Author: Not sure about "manually", but I think I rephrased this to solve what you're pointing out.
